Tools/Scripts/libraries/resultsdbpy/resultsdbpy/view/view_routes.py

```diff
@@ -42,7 +42,7 @@ def __init__(
         import_name=__name__,
         title='Results Database', auth_decorator=None,
         archive_routes=None,
-        suite_types=None,
+        suite_types=None, default_architecture=None,
```

Comment:
Nit: Can we move this to a new line to be consistent with most of the rest of the args here? (And consistent with [Black](https://github.com/psf/black)'s code style.)